packages/semantic-ui/package.json

```diff
@@ -50,7 +50,7 @@
     "@types/react-test-renderer": "^17.0.2",
     "atob": "^2.0.3",
     "dts-cli": "^1.6.0",
-    "eslint": "^8.26.0",
+    "eslint": "^8.23.0",
```

Comment:
Can you please revert this down-grade?

Reply:
eslint fails on the higher version, complaining about missing packages. Checked other themes use this version. keeping it consistent like other themes

Reply:
Ok, thanks!